plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowMavenModelTest.java

```diff
@@ -171,7 +171,7 @@ public void testXpathExpression() throws Exception {
         new ByteArrayInputStream(simplifiedModel.getBytes()));
 
     XPath realXpath = XPathFactory.newInstance().newXPath();
-    realXpath.setNamespaceContext(DataflowMavenModelFactory.POM_NS_CONTEXT);
+    realXpath.setNamespaceContext(DataflowMavenModelFactory.pomNamespaceContext);
```

Comment:
we should be able to use a separate namespace context object here; no real need to reach into the private parts

Reply:
I made the field private. I removed three tests too as a result, but those are the ones that I added in this PR.